File: VB/WebSite/App_Code/MySavedObjects.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.Web

Public Class MySavedObjects
    Public Sub New()
        _FileName = String.Empty
        _Url = String.Empty
    End Sub
    ' Fields...
    Private _FileName As String
    Private _Url As String
    Private _RowNumber As Integer

    Public Property RowNumber() As Integer
        Get
            Return _RowNumber
        End Get
        Set(ByVal value As Integer)
            _RowNumber = value
        End Set
    End Property

    Public Property Url() As String
        Get
            Return _Url
        End Get
        Set(ByVal value As String)
            _Url = value
        End Set
    End Property

    Public Property FileName() As String
        Get
            Return _FileName
        End Get
        Set(ByVal value As String)
            _FileName = value
        End Set
    End Property
End Class

File: CS/WebSite/App_Code/MySavedObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.Web;

public class MySavedObjects
{
    public MySavedObjects()
    {
        _FileName = String.Empty;
        _Url = String.Empty;
    }
    // Fields...
    private string _FileName;
    private string _Url;
    private int _RowNumber;

    public int RowNumber
    {
        get { return _RowNumber; }
        set { _RowNumber = value; }
    }

    public string Url
    {
        get { return _Url; }
        set { _Url = value; }
    }

    public string FileName
    {
        get { return _FileName; }
        set { _FileName = value; }
    }
}